core/core_integrations/discord.js

```diff
@@ -0,0 +1,103 @@
+var shim = require.once('../shim.js'),
+    discord = require('discord.js'),
```

Comment:
Given it isn't in `package.json`, have you tested when the dependency does not exist and has not been installed manually?